addons/account/tests/test_tour.py

```diff
@@ -22,18 +36,16 @@ def test_01_account_tour(self):
         account_with_taxes.write({
             'tax_ids': [Command.clear()],
         })
-        # This tour doesn't work with demo data on runbot
-        all_moves = self.env['account.move'].search([('move_type', '!=', 'entry')])
-        all_moves.button_draft()
-        all_moves.with_context(force_delete=True).unlink()
-
-        # In case of latam impacting multiple countries, disable the required fields manually.
-        if 'l10n_latam_use_documents' in self.env['account.journal']._fields:
-            self.env['account.journal']\
-                .search([('company_id', '=', self.env.company.id), ('type', '=', 'sale')])\
-                .write({'l10n_latam_use_documents': False})
 
         self.start_tour("/web", 'account_tour', login="admin")
 
     def test_01_account_tax_groups_tour(self):
+        product = self.env.ref('product.product_product_5')
+        new_tax = self.env['account.tax'].create({
+            'name': '10%',
```

Comment:
Name a bit too generic that might conflict with taxes in some l10n (unicity constraint)
Might be picked up only in single build nightly